docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1143,6 +1165,83 @@ See the :ref:`hive_examples` for more information.
+      ``s3a://test/name=${name}/``. Mapped from AWS Athena table property
+      `storage.location.template <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-setting-up.html#partition-projection-specifying-custom-s3-storage-locations>`_
+    - ``${table_location}/${partition_name}``
+
+.. _hive_column_properties:
+
+Column properties
+-----------------
+
+.. list-table:: Hive connector column properties
+  :widths: 20, 60, 20
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default
+  * - ``partition_projection_type``
+    - Defines the type of partition projection to be used on this column.
+      May be used only on partition columns. Available types:
+      ``ENUM``, ``INTEGER``, ``DATE``, ``INJECTED``.
+      Mapped from AWS Athena table property
+      `projection.${columnName}.type <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-supported-types.html>`_.
+    -
+  * - ``partition_projection_values``
+    - Used with ``partition_projection_type=ENUM``. Contains a static
+      list of values used to generate partitions.
+      Mapped from AWS Athena table property
+      `projection.${columnName}.values <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-supported-types.html>`_.
+    -
+  * - ``partition_projection_range``
+    - Used with ``partition_projection_type=INTEGER|DATA``. It is a two-element
+      array, describing range edges used to generate partitions.
+      Generation starts from the left bound then increments by the defined
+      ``partition_projection_interval`` and ends on the right bound,
+      Mapped from AWS Athena table property
+      `projection.${columnName}.range <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-supported-types.html>`_.
+    -
+  * - ``partition_projection_interval``
+    - Used with ``partition_projection_type=INTEGER|DATA``.
+      Represents the interval used to generate partitions within
+      the given range ``partition_projection_range``.
+      Mapped from AWS Athena table property
+      `projection.${columnName}.interval <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-supported-types.html>`_.
+    -
+  * - ``partition_projection_digits``
+    - Used with ``partition_projection_type=INTEGER``.
+      The number of digits to be used with integer column projection.
+      Mapped from AWS Athena table property
+      `projection.${columnName}.digits <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-supported-types.html>`_.
+    -
+  * - ``partition_projection_format``
+    - Used with ``partition_projection_type=DATE``.
+      The date column projection format.
+      Mapped from AWS Athena table property
+      `projection.${columnName}.format <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-supported-types.html>`_.
+    -
+  * - ``partition_projection_interval_unit``
+    - Used with ``partition_projection_type=DATA``.
```

Comment:
For here and all others.. should we have description first and then for what type a property applies?

Reply:
I think this order makes more sense because it's informing you whether or not you should keep reading. I don't care about `partition_projection_digits` if I have my type set to `DATE`, and so it's good for that information to be up front.

Reply:
+1 for @colebow

Reply:
Ok .. makes sense to me